### PR TITLE
Capstone-Project1 for Microverse Module 1

### PR DESCRIPTION
**Pull request by Jusha25**
# Capstone Project 1
## Build With--Html,CSS,Javascript, Linters.
 
This is my first Capstone Project which is a website for conferences. This website is responsive and is for both mobile and desktop.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @Jusha25, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter errors, kindly make sure all the checks pass

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Jusha25** [comment]
[Ismailco](https://github.com/Ismailco) Hi But I tried to remove it its is not solving can u please guide me in that.

**Comment by Ismailco** [comment]
@Jusha25, You need to use the correct indentation in the CSS file

File: style.css [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+  justify-content: center;
+  align-items: center;
+  color: white;
+}
+
+.summit-photo2 {
+  background-image:
+          linear-gradient(to right, rgba(151, 42, 22, 0.6), rgba(151, 42, 22, 0.6)),
+      url(images/meeting-2icon.jpg);
+}
```
Comment (Ismailco) [inline style.css:216]:
```
.summit-photo2 {
  background-image: linear-gradient(to right, rgba(151, 42, 22, 0.6), rgba(151, 42, 22, 0.6)), url(images/meeting-2icon.jpg);
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+.old-look {
+  font-size: 12px;
+  padding: 0 7%;
+}
+
+.summit-photo1 {
+  background-color: #972a16;
+  background-image:
+          linear-gradient(to right, rgba(151, 42, 22, 0.6), rgba(151, 42, 22, 0.6)),
+      url(images/meeting-1icon.jpg);
```
Comment (Ismailco) [inline style.css:199]:
```
background-image: linear-gradient(to right, rgba(151, 42, 22, 0.6), rgba(151, 42, 22, 0.6)), url(images/meeting-1icon.jpg);
```

Comment (Ismailco) [inline style.css]:
```suggestion
  background-image: linear-gradient(to right, rgba(151, 42, 22, 0.6), rgba(151, 42, 22, 0.6)), url(images/meeting-1icon.jpg);
```

File: style.css (hunk 1 above)
Comment (Ismailco) [inline style.css]:
```suggestion
.summit-photo2 {
  background-image: linear-gradient(to right, rgba(151, 42, 22, 0.6), rgba(151, 42, 22, 0.6)), url(images/meeting-2icon.jpg);
}
```

**Comment by Jusha25** [comment]
[Ismailco](https://github.com/Ismailco) I have done that. Please review now.

**Comment by Ismailco** [comment]
@Jusha25, request a review in the dashboard

File: README.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+## Setup
+
+To get a local copy up and running follow these simple example steps.
+
+To get access to this repository, move to "capstone-project" repository, and then either use [ssh] or [https] to clone onto your local machine. To clone checkout this documentation on github (https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
+## Live Demo
+
+https://jusha25.github.io/Capstone-Project1/
+
```
Comment (mikethreels) [inline README.md]: - [ ] the live demo link is not working as expected please fix this before submitting it for another review.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,608 @@
...
+  justify-content: center;
+  flex-direction: column;
+}
+
+.main-section ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+}
```
Comment (mikethreels) [inline style.css:101]: - [ ] [OPTIONAL] a lot of the code in your CSS file is repeated for example the display flex is repeated as you can see in this example, you could make your code a lot shorter and more readable if you made 1 class for flex-box for example.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Shaili, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vechicin — COMMENTED** [review]
# Hi @Jusha25 ✋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😞 

## Invalid Code Review Request ⚠️ 

You have submitted a project, that is not passing linters checks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Jusha25** [comment]
[vechicin](https://github.com/vechicin) Please do review now.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+      </div>
+    </section>
+    <footer id="join">
+      <img src="images/creative.svg" alt="" />
+      <div class="footright">
+        <p>2019 Town_Square,Washington,USA</p>
+        <!-- <br /> -->
+        <p>All Rights reserved</p>
+      </div>
+    </footer>
```
Comment (Reem-lab) [inline index.html:165]: - [ OPTIONAL ] I think the footer section is not in the home page for mobile version it is just in the about page so kindly remove it  from the mobile version it is just displayed in the desktop version so keep it there and make it display:none in the mobile version ✅💪🏻💪🏻

- Your home page
image

**Review by Reem-lab — APPROVED** [review]
# Hi @Jusha25, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥🔥🔥🔥🎉🎉 



## Highlights
All linters checks passed ✔️
Descriptive pull request ✔️
Good project implementation ✔️
Used Github flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Ismailco) [reply style.css]: ### This is not a review, it's just a comment to help @Jusha25, fix the linters

**Pull request merged**